core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,44 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $scope.userIsLoggedIn = GLOBALS.userIsLoggedIn;
+        $scope.activitiesOwned = {explorations: {}, collections: {}};
+        if ($scope.userIsLoggedIn) {
+          $http.get('/creatordashboardhandler/data')
+            .then(function(response) {
+              $scope.libraryGroups.forEach(function(activityGroups) {
```

Comment:
The dummy var here should be libraryGroup, right?

Reply:
Ah, maybe I should be more clear about the principle, so that you can apply it in the future.

If you have a list variable named "things" then the singular of that should be called "thing".

What you had was a variable named "libraryGroups" and you called the singular of that "activityGroups". That leads to confusion.